src/main/kotlin/app/revanced/patches/inshorts/ad/fingerprints/InshortsAdsFingerprint.kt

```diff
@@ -0,0 +1,17 @@
+package app.revanced.patches.inshorts.ad.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object InshortsAdsFingerprint : MethodFingerprint(
```

Comment:
Is the method this fingerprint resolves to an Inshort specific thing or a method shaded from the Google ads library? If it is the latter, the patch should be converted into a generic Google ads patch.